qemu/tests/cfg/blockdev_commit_query_named_block_nodes.cfg

```diff
@@ -0,0 +1,38 @@
+- blockdev_commit_query_named_block_nodes:
+    type = blockdev_commit_query_named_block_nodes
+    virt_test_type = qemu
+    only Linux
+    images += " data"
+    force_create_image_data = yes
+    force_remove_image_data = yes
+    start_vm = yes
+    kill_vm = yes
+    storage_pools = default
+    storage_type_default = "directory"
+    storage_pool = default
+    image_size_data = 500M
+    image_name_data = data
+    snapshot_tags = sn1 sn2 sn3 sn4
```

Comment:
what about snapshot_tags_data? of course, snapshot_tags works well, but snapshot_tags_data explicitly shows we make snapshots for data image

Reply:
Have updated.